src/argparse.cpp

```diff
@@ -15,6 +15,7 @@ const struct option long_options[] =
 {
    {"autocmd",  required_argument, nullptr, 'a'},
    {"cfg_file", required_argument, nullptr, 'c'},
+   {"greenmode",required_argument, nullptr, 'g'}, 
```

Comment:
This should be a value in the configuration file rather than a flag.

Reply:
Yes, would be better. I'm taking a look.